frontend/src/react-query/api/security.tsx

```diff
@@ -69,32 +54,22 @@ export const useListRolesQuery = (
 };
 
 export const useListRoleMembersQuery = (
-  input?: MessageInit<ListRoleMembersRequestDataPlane>,
+  input?: MessageInit<ListRoleMembersRequest>,
   options?: QueryOptions<GenMessage<ListRoleMembersRequest>, ListRoleMembersResponse>,
 ) => {
-  const listRoleMembersRequestDataPlane = create(ListRoleMembersRequestSchemaDataPlane, {
+  const listRoleMembersRequest = create(ListRoleMembersRequestSchema, {
     pageSize: MAX_PAGE_SIZE,
     pageToken: '',
     ...input,
   });
 
-  const listRolesMembersRequest = create(ListRoleMembersRequestSchema, {
-    request: listRoleMembersRequestDataPlane,
-  }) as MessageInit<ListRoleMembersRequest> & Required<Pick<MessageInit<ListRoleMembersRequest>, 'request'>>;
-
-  const listRoleMembersResult = useInfiniteQueryWithAllPages(listRoleMembers, listRolesMembersRequest, {
-    pageParamKey: 'request',
+  const listRoleMembersResult = useInfiniteQueryWithAllPages(listRoleMembers, listRoleMembersRequest, {
     enabled: options?.enabled,
-    getNextPageParam: (lastPage) =>
-      lastPage?.response?.nextPageToken
-        ? {
-            ...listRoleMembersRequestDataPlane,
-            pageToken: lastPage.response?.nextPageToken,
-          }
-        : undefined,
+    getNextPageParam: (lastPage) => lastPage?.nextPageToken || undefined,
+    pageParamKey: 'pageToken',
```

Comment:
Ae you sure this works well? I would be careful, make sure the infinite query works, try it out with page size of 1 and check that the next page gets retrieved after that.

Reply:
yes I did multiple checks